.github/workflows/build.yml

```diff
@@ -192,6 +217,48 @@ jobs:
             make CPPFLAGS='-Wall -Wextra -Werror'
             make check
 
+            make install "DESTDIR=`pwd`/install-dir"
+            maint/RunManifestTest install-dir maint/manifest-makeinstall-linux
+
+  distcheck:
+    name: Build & verify distribution
+    runs-on: ubuntu-24.04 # TODO: Update to ubuntu-latest when that switches to 24.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          submodules: true
+
+      - name: Prepare
+        run: |
+          echo "XXX UMASK"
+          umask
+          echo "XXX ls -l /usr/share/aclocal/"
+          ls -l /usr/share/aclocal/ || true
+          echo "XXX ls -l /usr/share/libtool/build-aux/"
+          ls -l /usr/share/libtool/build-aux/ || true
+          echo "XXX libtool"
+          which libtoolize
+          /usr/bin/libtoolize --version
+
+          ./autogen.sh
+
+          ls -l m4/
+
+          echo "XXX FIXING"
+          chmod ugo-x m4/* ltmain.sh
```

Comment:
I'm debugging something weird here.

On the GitHub Actions workflow, the files created by libtoolize are executable. They have some weird setup going on in a Docker container that they use to make their OS build? I can't work it out.

Basically some files are copied into our working tree from system directories, and they have permissions which are simply "wrong" in those system directories.

Reply:
I opened a ticket with GitHub, since it appears to be something strange on their end:
https://github.com/actions/runner-images/issues/11212